tests/test_cli.py

```diff
@@ -133,19 +134,16 @@ def test_train_command():
         result = runner.invoke(app, [
             "train",
             data_path,
-            model_path,
             label_binarizer_path,
+            model_path,
```

Comment:
did we change the order? this might affect `dvc.yaml`

Reply:
resolving :)

Reply:
you know what the weird thing is, dvc actually had the correct order, just the test had the wrong one. Meaning it was passing until now with the wrong order somehow

Reply:
@nsorros

Reply:
I don't get it, it shouldn't have based on what's in the train code.